### PR TITLE
update  README.md

### PR DESCRIPTION
**Pull request by binbinCheng**
**Update the content in the README document**

### CODE REVIEW

**Review by casionone — APPROVED** [review]
LGTM

File: README_CN.md [hunk 1]
```diff
@@ -66,37 +66,35 @@ Linkis 自 2019 年开源发布以来，已累计积累了 700 多家试验企
...
+- **全栈计算存储引擎架构支持**：  能够接收、执行和管理针对各种计算存储引擎的任务和请求，包括离线批量任务、交互式查询任务、实时流式任务和数据湖任务；
+- **统一上下文服务**：支持跨用户、系统、计算引擎去关联管理用户和系统的资源文件（JAR、ZIP、Properties 等），结果集、参数变量、函数、UDF等，一处设置，处处自动引用；
+- **统一物料**： 提供了系统和用户级物料管理，可分享和流转，跨用户、跨系统共享物料；
+- **统一数据源管理**：  提供了Hive、ElasticSearch、Mysql、Kafka、MongoDB 等类型数据源信息的增删查改、版本控制、连接测试和对应数据源的元数据信息查询能力；
+- **错误码能力**：提供了任务常见错误的错误码和解决方案，方便用户自助定位问题；
 
 # 支持的引擎类型
 
-| **引擎名** | **支持底层组件版本 <br/>(默认依赖版本)** | **Linkis 版本要求** | **是否默认包含在发布包中** | **说明** |
+| **引擎名** | **支持底层组件版本<br/>(默认依赖版本)** | **Linkis 1.X 版本要求** | **是否默认包含在发布包中** | **说明** |
```
Comment (casionone) [inline README_CN.md:80]: About **Linkis 版本要求** 
It is recommended to use version 1.0.3 not 1.0.0_rc1

**Review by casionone — CHANGES_REQUESTED** [review]
About **Linkis 版本要求** 
It is recommended to use version 1.0.3 not 1.0.0_rc1

File: README_CN.md (hunk 1 above)
Comment (binbinCheng) [reply README_CN.md:80]: already edited

**Review by peacewong — APPROVED** [review]
LGTM.

**Review by casionone — APPROVED** [review]

**Pull request merged**